web/src/ui_init.ts

```diff
@@ -254,8 +256,10 @@ export function initialize_kitchen_sink_stuff() {
         message_viewport.set_last_movement_direction(delta);
     }, 50);
 
+    // TODO(evy): Figure out why e.originalEvent isn't being typed
+    // as a WheelEvent
```

Comment:
[#frontend > typescript for wheel event @ 💬](https://chat.zulip.org/#narrow/channel/6-frontend/topic/typescript.20for.20wheel.20event/near/2264997)